### PR TITLE
added some mathml functions

### PR DESCRIPTION
**Pull request by shubhamabhang77**
#### References to other Issues or PRs
#16036 

#### Brief description of what is fixed or changed

- Added Lambda, ceiling, floor and exp functions to mathml printer

#### Release Notes

- printing

   - Added Lambda, floor, ceiling and exp functions to the mathml printer

### CODE REVIEW

**Comment by oscargus** [comment]
Thanks!

exp is already in #16126 (and that uses the special MathML character for exponential e).

For ceiling and floor there should not be more than one argument, so no need to loop.

Lambda will not print correctly as the bracket is around the lambda symbol only.

I think one only require `<mrow>` if there are several top-level tags, so when there is a bracket around everything there should be no need.

**Comment by shubhamabhang77** [comment]
```
	<mrow>
		<mrow>
			<mfenced>
				<mi>x</mi>
			</mfenced>
		</mrow>
		<mo>&#x21A6;</mo>
		<mrow>
			<mi>x</mi>
			<mo>+</mo>
			<mn>1</mn>
		</mrow>
	</mrow>
```

When I remove the first ```<mrow>``` extra ```,``` are getting added between ```Lambda``` and ```x```.
That's why I didn't remove <mrow>.
If you want to see how it renders without that first <mrow>, then you will able to see the difference.

File: sympy/printing/mathml.py [hunk 1]
```diff
@@ -1221,6 +1221,38 @@ def _print_EmptySet(self, e):
...
+        x.setAttribute('close', u'\u2309')
+        x.appendChild(self._print(e.args[0]))
+        mrow.appendChild(x)
+        return mrow
+
+
+    def _print_Lambda(self, e):
+        x = self.dom.createElement('mfenced')
+        mrow = self.dom.createElement('mrow')
+        mrow.appendChild(self._print(e.args[0]))
```
Comment (oscargus) [inline sympy/printing/mathml.py]: One could update the code similar to the LaTeX code that if there is only one symbol print it without the brackets:
```
        symbols = expr.args[0]
        if len(symbols) == 1:
            symbols = self._print(symbols[0])
        else:
            symbols = self._print(symbols)
        mrow.appendChild(symbols)
```

**Comment by oscargus** [comment]
Hmm, below is the first answer I wrote, but when I checked the code I am not so sure that the current Python code is the same that generates the MathML expression in the example. I think that the updated code for `_print_Lambda` is the correct one, but that the test should be updated to reflect that.

Also, can you please add a test for a lambda with two variables, like `Lambda((x, y), x + y)` to see how that prints.

With those tests (and possibly the change I added a comment in the code about) it should be good to go!

--- Old comments

Ah, I see the problem. I think the solution is to do something like

```
<mfenced>
<mrow>
print x
print arrow
print expression
</mrow>
</mfenced>
```
<mrow> works as a "bracket" to keep more than one tag in a single item. I didn't know that MathML always inserted commas though. I thought it only did it in e.g. <mfenced>. (I do not think there is a need for <mrow> for the x+1 expression either as there are operators inbetween the items, but not sure... Not an issue though, especially since it is some other code that adds it.)

The problem right now is that it is printed as

(x) -> x+1
instead of 
(x -> x+1)
which the outlined code above should do.

**Comment by shubhamabhang77** [comment]
Now is this how you wanted?

**Comment by oscargus** [comment]
Yes, that looks both good and consistent with the LaTeX output. Thanks!

**Comment by shubhamabhang77** [comment]
Thanks to you as well.

**Pull request merged**